src/styles/types.js

```diff
@@ -5,6 +5,7 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
+import * as React from 'react';
```

Comment:
Good catch!  As per the [Flow React documentation](https://flow.org/en/docs/react/types/), React types must be imported in files where they are used.

`import * as React from 'react'` is shorthand for including access to all types.

Without this, the `React.Node` usages below would resolve to `any` which potentially results in a lot of uncovered code.

Reply:
Thanks! What do you think of the change from `Overflow?: React.Node,` to `Overflow?: typeof React.Component,` I can't seem to find any guides that suggest this is a good way. It just happened to work for me.

Reply:
Ah, added a comment below before I saw this.  I don't have a lot of context on this code, but from a quick glance it appears that there are corresponding concrete implementations for of these icons.

For example, `ArrowDown`:
https://github.com/uber-web/baseui/blob/73fc65624f2cbf067bd731c4e5af9faf1eeacc4f/src/icon/arrow-down.js#L15-L37

My assumption would be that the props passed into `ArrowDown` should always match those passed into `theme.icons.ArrowDown`.  If that's the case, we could use the same `IconPropsT` type.  Something like:

```
import type {IconPropsT} from '<some_path_to_type>';
...
export type IconT = {
   ...
   ArrowDown?: React.ComponentType<IconPropsT>,
   ...
};
```